internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/integration/utils/VeniceRouterWrapper.java

```diff
@@ -145,7 +145,7 @@ static StatefulServiceProvider<VeniceRouterWrapper> generateService(
           .put(ROUTER_HTTPASYNCCLIENT_CONNECTION_WARMING_LOW_WATER_MARK, 1)
           .put(ROUTER_MAX_OUTGOING_CONNECTION, 10)
           // To speed up test
-          .put(ROUTER_NETTY_GRACEFUL_SHUTDOWN_PERIOD_SECONDS, 0)
+          .put(ROUTER_NETTY_GRACEFUL_SHUTDOWN_PERIOD_SECONDS, 10)
```

Comment:
Will this change slowdown the existing integration tests greatly?
We don't need to do graceful shutdown in the integration test env unless the test is validating this particular feature, right? When there is a need, the test can override this config.

Reply:
Sure, I can override this for the failing tests.